modin/pandas/base.py

```diff
@@ -317,55 +426,63 @@ def _default_to_pandas(self, op, *args, **kwargs):
                 return result
 
     def _get_axis_number(self, axis):
+        """
+        Implement [METHOD_NAME].
+
+        TODO: Add more details for this docstring template.
+
+        Parameters
+        ----------
+        What arguments does this function have.
+        [
+        PARAMETER_NAME: PARAMETERS TYPES
+            Description.
+        ]
+
+        Returns
+        -------
+        What this returns (if anything)
+        """
         return (
             getattr(pandas, type(self).__name__)()._get_axis_number(axis)
             if axis is not None
             else 0
         )
 
     def __constructor__(self, *args, **kwargs):
```

Comment:
This constructs `DataFrame` or `Series`object , not `BasePandasDataset` object. So, a docstring should be as follow:
```
"""Constructs DataFrame or Series object in depend of self's type"""
```